interpreter/cling/lib/Interpreter/CIFactory.cpp

```diff
@@ -873,14 +873,16 @@ static void stringifyPreprocSetting(PreprocessorOptions& PPOpts,
   }
 
   static llvm::IntrusiveRefCntPtr<DiagnosticsEngine>
-  SetupDiagnostics(DiagnosticOptions& DiagOpts) {
+  SetupDiagnostics(DiagnosticOptions& DiagOpts, std::string ExeName) {
```

Comment:
Can we pass this by const ref:
```suggestion
  SetupDiagnostics(DiagnosticOptions& DiagOpts, const std::string &ExeName) {
```